meson.build

```diff
@@ -244,6 +244,11 @@ if host_os_family == 'linux' or host_os_family == 'qnx'
   else
     extra_libs_private += ['-ldwarf', '-lelf', '-lz']
   endif
+  if have_shared_glib
+    dwarf_dep = dependency('libdw')
+    elf_dep = dependency('libelf')
```

Comment:
I wonder if the cleanest solution here is to fix our build recipes (in `releng/deps.mk`) so these two come with `.pc`-files in our SDK -- that way we can drop our ugly hacks here and simply use `dependency()` unconditionally. What do you think?

Reply:
That sounds much better~ I'll take a look at this instead